### PR TITLE
Corrected partial grammatical mistake

### PR DESCRIPTION
**Pull request by vatsalvatsyayan**

### CODE REVIEW

**Review by MasterWayZ — APPROVED** [review]
Good catch!

File: README [hunk 1]
```diff
@@ -1,7 +1,7 @@
 Bugzilla
 ========
 
-Bugzilla is free and open source web-based bug-tracking software that is
+Bugzilla is a free and open source web-based bug-tracking software that is
```
Comment (CyberShadow) [inline README:4]: This does not look right. "Software" is an uncountable noun, isn't it?

Comment (vatsalvatsyayan) [reply README:4]: The subject of the matter is Bugzilla in this particular case

Comment (justdave) [reply README:4]: But the "a" isn't in front of "bugzilla," it's in front of "software."  @CyberShadow is correct, software is an uncountable noun, this was correct as-is.

**Comment by justdave** [comment]
For more information see https://en.m.wiktionary.org/wiki/software in the Usage notes section under "Noun"

**Pull request closed without merging**